scripts/debug_tools/reduce.py

```diff
@@ -0,0 +1,304 @@
+import argparse
+import json
+import operator
+import os
+import re
+import shutil
+import stat
+import subprocess
+import sys
+import tempfile
+
+
+def check_files_existence(report_dir):
+    '''
```

Comment:
**Triple double-quoted** string should be used for docstring. My editor (pycharm) gives me this warning. Please replace this everywhere in this file.